templates/vue/src/components/Lightbox/media/MultiContentMedia/index.vue

```diff
@@ -310,11 +310,14 @@ button[disabled] {
 }
 
 .title {
-  color: #fff;
+  color: #111;
   margin-bottom: 1em;
 }
 
 .media-container {
+  background: #eee;
+  background-color: #eee;
+  color: #111;
```

Comment:
Again, not sure if we need this on a child element - can we keep it on the parent elements instead? Is there an overwrite somewhere that should be removed instead?

Reply:
Yep, I just checked, this rule is redundant... the ultimate element .media-wrapper already has its own rule